### PR TITLE
Netlify middleware

### PR DESCRIPTION
**Pull request by nnelgxorz**
# What is it?

- [x] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [x] My code follows the [developer guidelines of this project](../CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: scripts/qwik-city.ts [hunk 1]
```diff
@@ -38,6 +38,9 @@ export async function buildQwikCity(config: BuildConfig) {
       './middleware/express': {
         import: './middleware/express/index.mjs',
         require: './middleware/express/index.cjs',
+      },
+        './middleware/netlify-edge': {
+          import: './middleware/netlify-edge/index.mjs',
```
Comment (antoinepairet) [inline scripts/qwik-city.ts:43]: The formatting seems wrong
```suggestion
      './middleware/netlify-edge': {
        import: './middleware/netlify-edge/index.mjs',
```

**Review by antoinepairet — APPROVED** [review]
Reviewed the code, LGTM
Did not test it as I do not use Netlify

File: scripts/qwik-city.ts (hunk 1 above)
Comment (nnelgxorz) [reply scripts/qwik-city.ts]: Good catch!

**Comment by manucorporat** [comment]
cc @nnelgxorz could you try to merge main and run `yarn api.update` ?

**Comment by nnelgxorz** [comment]
Done!

**Pull request merged**